auth/user.ts

```diff
@@ -17,7 +17,26 @@ export interface UserSchema {
   tenantId: string | null;
 }
 
+export class UserInfo implements firebase.UserInfo {
+  readonly displayName: string | null;
+  readonly email: string | null;
+  readonly phoneNumber: string | null;
+  readonly photoURL: string | null;
+
+  constructor(
+    readonly uid: string,
+    readonly providerId: string,
+    rest: Partial<Omit<firebase.UserInfo, "uid" | "providerId">>
+  ) {
+    this.displayName = rest.displayName || null;
+    this.email = rest.email || null;
+    this.phoneNumber = rest.phoneNumber || null;
+    this.photoURL = rest.photoURL || null;
```

Comment:
Are we attempt to catch empty string values here as well?
If not we may want to use null coalescing rather then short circuiting

Reply:
Good catch! Will change.